lzy/allocator/src/main/resources/db/allocator/migrations/V10__operation_task.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TYPE task_status AS ENUM ('PENDING', 'RUNNING', 'FAILED', 'FINISHED', 'STALE');
+
+CREATE TYPE task_type AS ENUM ('UNMOUNT', 'MOUNT');
+
+CREATE TABLE IF NOT EXISTS operation_task(
```

Comment:
The main idea is to provide DB as a single source of truth about order of task execution.
Here's short explaination of `operation_task` fields:
- `id` - is bigserial and thus generated on insert of task. This is the main way to present order among certain tasks (see `entity_id`).
- `name` - for debug and readability purposes
- `entity_id` - this is the way to group tasks by some user-generated text id. Tasks with same `entity_id` are executed subsequently according the `id` field (in ascending order). Thus, task with smaller id will be executed first. Tasks with different `entity_id` can be executed in parallel.
- `type` - is necessary to match code representation of a task
- `status` - status of a task.
- `created_at`, `updated_at` - self-explainatory, for debug purposes
- `metadata` - JSON to keep task arguments and other useful information about the task. The content of this field is defined by user and parsed mainly depending by the type.
- `operation_id` - an operation that is linked to a task. Contains all details about execution. There should be `(0-1) <-> 1` relation between a task and an operation.
- `worker_id` - name of the instance that captured a task. This is needed to ensure that a task is executed just once.
- `lease_till` - deadline for scheduler instance to execute this task. Scheduler instance should update `lease_till` field. In case of instance death or any other reason that make instance impossible to finish a task, another scheduler instance can "capture" the task with expired `lease_till` deadline and replace `worker_id` field.